examples/evm/its-canonical-token/README.md

```diff
@@ -0,0 +1,54 @@
+# Call Contract Example
```

Comment:
This doesn't look right. I think this needs to get updated for this example.

Reply:
Forgot to fix the readmes. will fix